src/App.tsx

```diff
@@ -1,26 +1,279 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { createTodo, deleteTodo, getTodos, updateTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { TodoItem } from './TodoItem';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Filter>(Filter.all);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [processingIds, setProcessingIds] = useState<number[]>([]);
+  const [editId, setEditId] = useState<number | null>(null);
```

Comment:
you can create boolean state inside `TodoItem` component (it's related to the item only) and remove here